src/RecyclableMemoryStream.cs

```diff
@@ -706,7 +701,7 @@ private ArraySegment<byte> GetWritableBuffer(int minimumBufferSize)
             this.CheckDisposed();
             if (minimumBufferSize < 0)
             {
-                throw new ArgumentOutOfRangeException("sizeHint", $"sizeHint must be non-negative");
+                throw new ArgumentOutOfRangeException(nameof(minimumBufferSize), $"{nameof(minimumBufferSize)} must be non-negative.");
```

Comment:
The parameter name exposed to callers is `sizeHint`, not `minimumBufferSize`.

Reply:
Ah, you're right. I've refactored this place a bit.

Reply:
Let's simplify a bit:

```csharp
private ArraySegment<byte> GetWritableBuffer(int sizeHint)
{
    this.CheckDisposed();
    if (sizeHint< 0)
    {
        throw new ArgumentOutOfRangeException(nameof(sizeHint), $"{nameof(sizeHint)} must be non-negative.");
    }
    int minimumBufferSize = Math.Max(sizeHint, 1);

    // ...
}
```

Reply:
Sure, done.